chia/cmds/farm_funcs.py

```diff
@@ -239,7 +243,10 @@ def process_harvesters(harvester_peers_in: dict):
             print(f"Local Harvester{'s' if len(harvesters_local) > 1 else ''}")
             process_harvesters(harvesters_local)
         for harvester_ip, harvester_peers in harvesters_remote.items():
-            print(f"Remote Harvester{'s' if len(harvester_peers) > 1 else ''} for IP: {harvester_ip}")
+            if harvesters_hostnames[harvester_ip]:
```

Comment:
```suggestion
            if harvester_ip in harvesters_hostnames:
```
To avoid crashing if the lookup fails since `harvester_ip` then won't be in `harvesters_hostnames` and this would trigger an key error.